lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -418,10 +418,10 @@ def add_3ds(post, options)
             post[:additionalData][:scaExemption] = options[:sca_exemption] if options[:sca_exemption]
           end
         else
-          return unless options[:execute_threed] || options[:threed_dynamic]
+          return unless !options[:execute_threed].nil? || !options[:threed_dynamic].nil?
```

Comment:
`browserInfo` should only be included if execute_threed or threed_dynamic is true:
- `browserInfo` is required if `executeThreeD` is passed as true
- the inclusion of `browserInfo` is what tells Adyen to check Dynamic 3DS rules and maybe perform 3DS https://docs.adyen.com/risk-management/dynamic-3d-secure#page-introduction

Reply:
Alright, I think I've got what we want here. I was mostly following this https://docs.adyen.com/payments-fundamentals/psd2-sca-compliance-and-implementation-guide#specifypreferenceinyourapirequest for `executeThreeD` situations.